cluster-up

```diff
@@ -0,0 +1 @@
+kubevirtci/cluster-up
```

Comment:
If we must have an intermediate `kubevirtci` directory, we should probably just point the `Makefile` at that location rather than create symlinks

Reply:
pushed a change to remove the symlinks.